src/ape_ethereum/multicall/handlers.py

```diff
@@ -152,20 +168,43 @@ def add(self, call: ContractMethodHandler, *args, **kwargs):
 
     @property
     def returnData(self) -> List[HexBytes]:
-        if not self._result:
+        result = self._result  # Decl for typing reasons.
+        if not result:
             raise NotExecutedError()
 
-        elif isinstance(self._result, tuple):
```

Comment:
im not 100% but it seems like it was checking for one type but handling the other type, so i broke it into 2 checks to handle each `Tuple[int, List[HexBytes]]` and `List[Tuple[bool, HexBytes]]` respectively and now each should work, but this is the part i need the most testing help with.

The last case though still is not really annotated in the `_result` type - the one that is a Struct object

Reply:
OK I actually figured out what was going now. the bug only happens when using a single call in a multi-call, the results were already uplifted in a sense, so with some extra handling there, we are good to go and i fixed by mistake comments. 👍🏻